autopts/utils.py

```diff
@@ -477,6 +479,61 @@ def log_memory_usage():
     logging.debug(f"Memory usage: {mem_usage:.2f} MB")
 
 
+def extract_wid_testcases_to_csv():
+    log_dir = "./logs"
+    output_csv_path = "./wid_usage_report.csv"
+    profile_wid_map = defaultdict(lambda: defaultdict(set))
+
+    for root, _, files in os.walk(log_dir):
+        for filename in files:
+            if filename.endswith(".log"):
```

Comment:
Consider replacing this with 

```suggestion
    for log_file in Path("./logs").rglob("*.log"):
```

To reduce the complexity and number of indendations

(Generally I would recommend using Pathlib over os.path for nearly everything)